### ISSUE TITLE
Implement a scanning mode

### ISSUE THREAD

**Issue by merbanan** [merbanan/rtl_433#1885]
In this scanning mode iterate over a parameter range and after a time duration output the "best" setting based on some kind of metric.

The initial thought is to just do a 100kHz step of the frequency parameter and then report the amount of received messages.

This would help out to find a reasonable default for users that might work as well as looking a spectrograms etc.

**Comment by zuckschwerdt** [comment]
Quick and dirty: maybe just run a frequency hopping then sort through the output.

**Comment by merbanan** [comment]
Yes, that is my first thought also. I would just like for it to be extendable. And I would like to have the feature of selecting the "best" parameters after the scanning is done also. And a delayed start of the scanning to warm up the components etc.

**Comment by zuckschwerdt** [comment]
This might be a good case for a separate binary. We could polish the "API" along the way.
